doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -458,7 +459,7 @@ Launch the RTVM
 
   .. code-block:: none
 
-     $ sudo cp /home/acrn/work/acrn-hyperviso/misc/vm_configs/sample_launch_scripts/nuc/launch_hard_rt_vm.sh  /usr/share/acrn/
+     $ sudo cp /home/acrn/work/acrn-hyperviso/misc/config_tools/data/sample_launch_scripts/nuc/launch_hard_rt_vm.sh  /usr/share/acrn/
```

Comment:
This script is a little confusing because one of the comment indicates it is for a KBL NUC (see [here](https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/data/sample_launch_scripts/nuc/launch_hard_rt_vm.sh#L4) but the NVMe PCI BDF is the one for the WHL platform. I think we should do two things:
* Modify the comment in the script to match the BDF in there
* Add a note to this guide that if using a KBL NUC, the script **must** be adapted to match the BDF on the actual HW platform

Reply:
add note in the latest commit: 9bcd1c55dbc4b8f2e289ec76213f45365c6a78a5

Reply:
Can you also modify the comment [here](https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/data/sample_launch_scripts/nuc/launch_hard_rt_vm.sh#L4) wrongly indicating this script is for a KBL NUC? Thanks

Reply:
ok